src/hotspot/os_cpu/bsd_aarch64/vm_version_bsd_aarch64.cpp

```diff
@@ -188,29 +232,30 @@ unsigned long VM_Version::os_get_processor_features() {
   id_aa64pfr0 = READ_SPECIALREG(ID_AA64PFR0_EL1);
 
   if (ID_AA64ISAR0_AES(id_aa64isar0) == ID_AA64ISAR0_AES_BASE) {
-    auxv = auxv | HWCAP_AES;
+    auxv = auxv | CPU_AES;
   }
 
   if (ID_AA64ISAR0_AES(id_aa64isar0) == ID_AA64ISAR0_AES_PMULL) {
-    auxv = auxv | HWCAP_PMULL;
+    auxv = auxv | CPU_PMULL;
```

Comment:
The intention was to use the HWCAP_* defines that were put at the top of the file. Please put these back to HWCAP_*.

Reply:
you didn't miss to tell me that and the intention was clear :-) it`s restored in last commit

Reply:
Thanks. I still see several lines changed from HWCAP_* to CPU_*. Did you push your last commit up?

Reply:
<<Thanks. I still see several lines changed from HWCAP_* to CPU_*. Did you push your last commit up?>>

is  https://github.com/battleblow/openjdk-jdk11u/pull/67/commits/45304ad59a24469cb65c7023bb9270a0f0c29a5e not pushed , can`t you see it?
afaik git Recently  automatically pushes commits ? or do I have to do something explicitly

Reply:
I see that commit. Please take a look at the output from
`git diff bsd-port -- src/hotspot/os_cpu/bsd_aarch64/vm_version_bsd_aarch64.cpp` 
to see if it matches what github is showing me in the pull request.